### PR TITLE
Adhere to scala inclusive language guide and fix typos

### PR DESCRIPTION
**Pull request by erikvanoosten**
Words like 'easy' and 'effortlessly' should not be used. See https://docs.scala-lang.org/contribute/inclusive-language-guide.html#the-words-easy-simple-quick-and-trivial for more background.

Also: fix some small typos.

### CODE REVIEW

File: docs/tutorials/testing-http-apps.md [hunk 1]
```diff
@@ -82,12 +80,12 @@ object TestUsingTestClient extends ZIOSpecDefault {
 
 ## TestServer
 
-Using the `TestServer` we can write tests for our HTTP applications by starting a live server instance on the localhost.
+WIth the `TestServer` we can write tests for our HTTP applications by starting a live server instance on the localhost.
```
Comment (hearnadam) [inline docs/tutorials/testing-http-apps.md]: `s/WIth/With/`

Comment (erikvanoosten) [reply docs/tutorials/testing-http-apps.md]: Oops, thanks. Fixed.

**Comment by jdegoes** [comment]
I encourage all contributions focused on improving spelling and grammar, improving structure, and adding or enhancing existing explanations. But contributions focused on language policing -- such as banning the word 'easy' because it might trigger some people -- are not the sort of contributions this project encourages.

Please separate out the typo fixes and they will be gladly merged! 🙏

**Comment by erikvanoosten** [comment]
I thought the same. Then I read the guidelines and changed my mind. But its your show. Feel free to extract the typos and use it anyway you like.

**Comment by erikvanoosten** [comment]
@jdegoes I was quite angry when I read your response. Which is ironic, because (now that I am cool again, really, its fine!) want to argue that this is not about triggering people at all. Its about being precise, clear and informative. To illustrate, when someone writes "this system supports large messages", almost no information is given at all. What is a large message? 1kB? 10GB? Some people will investigate further, but many will fill in some size they consider 'large' and perhaps later be disappointed when they find out their assumption was totally off. So I hope you agree that writing "large message" without any further indication of what "large" means is sub-optimal. Now I would like to argue that the exact same thing applies to the word "easy". "Easy" means different things to different people. The proposed text changes in this PR address this by either removing the word, or by comparing the 'easiness' to other solutions. The result IMHO is a better text for everybody.

**Pull request closed without merging**